docs/kb/intro-about.md

```diff
@@ -52,4 +52,4 @@ VMware Event Broker Appliance enables customers to quickly get started with pre-
 
 ### Community Use Cases
 
-Please see [this list here](https://github.com/vmware-samples/vcenter-event-broker-appliance/USECASES.md) for a collection of use cases from the VMware Event Broker Appliance community.
\ No newline at end of file
+Please see [this list here](https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/development/USECASES.md) for a collection of use cases from the VMware Event Broker Appliance community.
```

Comment:
question: shouldn't this be `master` branch?

Reply:
Indeed it should.